odilia/src/state.rs

```diff
@@ -59,57 +49,6 @@ impl ScreenReaderState {
 
 		let mode = Mutex::new(ScreenReaderMode { name: "CommandMode".to_string() });
 
-		tracing::debug!("Reading configuration");
```

Comment:
Good job removing it from the state initializer. I think it was my mistake that let it in here in the first place.